tests/e2e/cucumber/steps/ui/shares.ts

```diff
@@ -321,3 +321,30 @@ Then(
     expect(actualMessage).toBe(message)
   }
 )
+
+Then(
+  /^"([^"]*)" (should|should not) be able to manage share with user "([^"]*)"$/,
+  async function (
+    this: World,
+    stepUser: any,
+    actionType: string,
+    recipient: string
+  ): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const shareObject = new objects.applicationFiles.Share({ page })
+    const changeRole = shareObject.changeRoleLocator(
+      this.usersEnvironment.getUser({ key: recipient })
+    )
+    const changeShare = shareObject.changeShareLocator(
+      this.usersEnvironment.getUser({ key: recipient })
+    )
+
+    if (actionType === 'should') {
+      await expect(changeRole).not.toBeDisabled
+      await expect(changeShare).not.toBeDisabled
```

Comment:
The lint problem was here
```suggestion
      await expect(changeRole).not.toBeDisabled()
      await expect(changeShare).not.toBeDisabled()
```